tests/test_examples.py

```diff
@@ -420,7 +420,7 @@ def test_simsopt_QH_comparison():
     nfp = 4
     aspect_target = 8.0
     # Initial (m=0, n=nfp) mode of the axis:
-    Delta = 0.2
+    Delta = 0.4
```

Comment:
I would recommend changing this variable name to `torsion`, since that is what this is controlling. `Delta` makes me think of the VMEC input parameter with the same name.

Reply:
Whatever name you decide, it would be useful to make it consistent with the Frenet-Serret notation. For example, torsion is tau in Frenet-Serret.